packages/demo-gatsby/src/templates/blog-post.js

```diff
@@ -113,15 +120,33 @@ function BlogPostTemplate(props) {
         <button onClick={() => setIsEditing(p => !p)}>
           {isEditing ? "Stop Editing" : "Start Editing"}
         </button>
+
         <TinaField
           name="rawFrontmatter.draft"
           Component={MyToggle}
-          type="checkbox"
+          type="toggle"
```

Comment:
This isn't the name of the component, it's a prop that gets passed to `Field` in `react-final-form`. It's necessary for the toggle field to work properly as a checkbox. 

Scroll to the bottom to read the [`value`](https://final-form.org/docs/react-final-form/types/FieldProps) section.